src/main/resources/org/datadog/jenkins/plugins/datadog/DataDogJobProperty/config.jelly

```diff
@@ -0,0 +1,20 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:section title="DataDog Tagging"> 
+        <f:optionalBlock name="enableFile" checked="${!instance.isTagFileEmpty()}" title="File from workspace" inline="true">
```

Comment:
I'm thinking that the options could have better names. Maybe the title of this one could become `Add tags from file in workspace`.